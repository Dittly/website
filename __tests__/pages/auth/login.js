/* eslint-env jest */
jest.mock('react-ga')

import React from 'react'
import renderer from 'react-test-renderer'
import Login from '../../../pages/auth/login'

describe('Login page', () => {
  it('renders / correctly', () => {
    const tree = renderer.create(
      <Login />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
