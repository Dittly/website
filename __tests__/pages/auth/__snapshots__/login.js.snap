// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page renders / correctly 1`] = `
<div>
  <div
    className="box-s1qf175f-0 wVoOR"
    color="white"
  >
    <a
      href={true}
      onClick={[Function]}
    >
      Dittly
    </a>
     | 
    <a
      href={true}
      onClick={[Function]}
    >
      Login
    </a>
  </div>
  <h1>
    Login
  </h1>
</div>
`;
